roles/setup_pgbackrest/tasks/validate_setup_pgbackrest.yml

```diff
@@ -14,10 +14,14 @@
   register: pgbackrest_config_check_res
   changed_when: pgbackrest_config_check_res.rc == '0'
 
+- name: Debug
+  ansible.builtin.debug:
+    var: "{{ pgbackrest_config_check_res.stdout_lines }}"
+
```

Comment:
Is there a particular reason for including the "stdout_line" in the debug? Do you think that including it could potentially expose sensitive information to the log? If we need to debug output, I believe a similar result is achievable with the "-vvv" option.

Reply:
This was added to debug errors thrown, I will remove